moderation-commands/clean.md

```diff
@@ -25,25 +25,24 @@ For more information on editing permission requirements for specific users/roles
 
 ### Options
 
-* `amount`: The amount of messages you want to clean (The default is 100)
+* `amount`: The amount of messages you want to clean (The default/maximum amount is 100 messages)
 * `user`: The user you want to do delete the messages of
 * `starts-with`: Delete messages that start with this value
 * `ends-with`: Delete messages that end with this value
 * `contains`: Delete messages that contain this value
-* `bots`: Whether or not to delete messages sent by bots (The default is false)
+* `bots`: Whether or not to delete messages sent by bots
```

Comment:
This still needs to include the same information I linked in the previous comment